doc/mf6io/mf6ivar/dfn/gwe-lke.dfn

```diff
@@ -294,7 +294,7 @@ tagged false
 in_record true
 reader urword
 longname boundary thermal conductivity
-description is the thermal conductivity of the of the interface between the aquifer cell and the lake.  
+description is the thermal conductivity of the interface between the aquifer cell and the lake.  
```

Comment:
I wonder if we can think of something other than interface (material?) for this description.  There is an associated width of this material, correct?  How is that width determined?  Is it a half cell thickness in either the vertical or horizontal direction depending on the connection type?

Reply:
In response, tweaked the explanations with f914f2b